website/content/docs/nia/installation/run.mdx

```diff
@@ -27,3 +27,12 @@ description: >-
   ```shell-session
   $ curl localhost:8558/status/tasks
   ```
+
+## Other Run modes
```

Comment:
I'm a bit split on where this should be, here or within the [CLI modes](https://www.consul.io/docs/nia/cli/cli-overview#modes) page. My preference would be the latter. I do see value in threading the learn guides within the "Get Started" navigation section, so maybe both location?

Reply:
I voluntarily didn't mess up with the CLI and API docs.

I am adding the intro link to the install page, so ideally they could find it as soon as they get interested in getting hands on and use the katacoda lab for testing.

Reply:
those are good places to add!